eng/common/build.ps1

```diff
@@ -150,4 +154,4 @@ catch {
   ExitWithExitCode 1
 }
 
-ExitWithExitCode 0
+ExitWithExitCode 0
```

Comment:
Please try not to remove trailing newlines. The majority of devs' editors seem to either not have that configured or _add_ them automatically.